aten/src/THC/THCApply.cuh

```diff
@@ -119,17 +120,18 @@ template <typename Op,
 __launch_bounds__(THC_APPLY_THREADS_PER_BLOCK, THC_APPLY_BLOCKS_PER_SM)
```

Comment:
these bounds can probably just be removed, they were a pretty old hack to get better occupancy on CC 3.5 (K40 generation).

Reply:
Done.